domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/fradrag/FradragStrategy.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.su.se.bakover.domain.beregning.fradrag
+
+import no.nav.su.se.bakover.domain.beregning.Fradrag
+import no.nav.su.se.bakover.domain.beregning.Fradragstype
+
+sealed class FradragStrategy {
```

Comment:
Är det forskjell på fradragen basert på om søker har ektefelle, etc.? Skjønner ikke helt forskjellen härifrån. 😊

Reply:
Inntil videre er det ingen forskjell (ikke støtte for dette enda), og alle variantene håndteres som de har gjort frem til nå. De ulike variantene vil kunne implementere ulike algoritmer for å beregne fradrag etterhvert.